src/ClientInterface.php

```diff
@@ -25,154 +26,154 @@
  * and more friendly interface to ease programming which is described in the
  * following list of methods:
  *
- * @method int         del(array|string $keys)
- * @method string|null dump($key)
- * @method int         exists($key)
- * @method int         expire($key, $seconds)
- * @method int         expireat($key, $timestamp)
- * @method array       keys($pattern)
- * @method int         move($key, $db)
- * @method mixed       object($subcommand, $key)
- * @method int         persist($key)
- * @method int         pexpire($key, $milliseconds)
- * @method int         pexpireat($key, $timestamp)
- * @method int         pttl($key)
- * @method string|null randomkey()
- * @method mixed       rename($key, $target)
- * @method int         renamenx($key, $target)
- * @method array       scan($cursor, array $options = null)
- * @method array       sort($key, array $options = null)
- * @method int         ttl($key)
- * @method mixed       type($key)
- * @method int         append($key, $value)
- * @method int         bitcount($key, $start = null, $end = null)
- * @method int         bitop($operation, $destkey, $key)
- * @method array|null  bitfield($key, $subcommand, ...$subcommandArg)
- * @method int         bitpos($key, $bit, $start = null, $end = null)
- * @method int         decr($key)
- * @method int         decrby($key, $decrement)
- * @method string|null get($key)
```

Comment:
Status can be returned in some cases, and Error IMHO when the exception behaviour is supressed.